google/datalab/contrib/mltoolbox/commands/_ml.py

```diff
@@ -330,6 +366,44 @@ def _transform(args, cell):
   _shell_process.run_and_monitor(cmd_args, os.getpid(), cwd=MLTOOLBOX_CODE_PATH)
 
 
+def _train(args, cell):
+  env = google.datalab.utils.commands.notebook_environment()
+  cell_data = google.datalab.utils.commands.parse_config(cell, env)
+  google.datalab.utils.commands.validate_config(
+      cell_data,
+      required_keys=['training_data', 'evaluation_data'],
+      optional_keys=['model_args'])
+
+  cmd_args = ['python', '-m', 'trainer.task',
+              '--job-dir', args['output_dir'],
+              '--analysis-output-dir', args['analysis_output_dir']]
+
+  def _process_train_eval_data(data, arg_name, cmd_args):
+    if isinstance(data, dict):
+      if 'csv_file_pattern' in data:
+        cmd_args.extend([arg_name, data['csv_file_pattern']])
+        cmd_args.append('--run-transforms')
+      elif 'transformed_file_pattern' in data:
+        cmd_args.extend([arg_name, data['transformed_file_pattern']])
+      else:
+        raise ValueError('Invalid training_data dict. ' +
+                         'Requires either "csv_file_pattern" or "transformed_file_pattern".')
+    elif isinstance(data, google.datalab.ml.CsvDataSet):
+      cmd_args.extend([arg_name, data.input_files[0]])
```

Comment:
This can be replaced with

for file_name in data.input_files:
  cmd_args.append(arg_name + '='+ file_name)

note 
1) more than one file can be given to the trainer
2)  We have to use '=' as the file name can have wildcard.